dcat/index.html

```diff
@@ -675,8 +675,8 @@ <h3>RDF representation</h3>
             The axiomatization of DCAT 2014 used global domain and range constraints for many of the properties defined in the DCAT namespace [[?VOCAB-DCAT-20140116]]. This makes quite strong ontological commitments, some of which are now being reconsidered - see individual issues noted inline below.
         </p-->
 
-        <p>The (revised) DCAT vocabulary is <a href="https://github.com/w3c/dxwg/tree/gh-pages/dcat/rdf/">available in RDF</a>.
-            The primary artefact <a href="https://w3c.github.io/dxwg/dcat/rdf/dcat2.ttl"><code>dcat2.ttl</code></a> is a serialization of the core DCAT vocabulary.
+        <p>The (revised) DCAT vocabulary is <a href="https://www.w3.org/ns/dcat2.rdf">available in RDF</a>.
```

Comment:
This comment implements the option suggested by @andrea-perego,  pointing to the namespace instead of the HTML can be even better than my previous proposal. In the end, the link https://www.w3.org/ns/dcat# brings to the same HTML I was proposing, but it is more elegant and flexible as it promotes the namespace on which we have deployed content negotiation.

```suggestion
        <p>The (revised) DCAT vocabulary is <a href="https://www.w3.org/ns/dcat#">available in RDF</a>.
```

Reply:
Thanks, @riccardoAlbertoni .